doc/whats_new.rst

```diff
@@ -55,7 +55,8 @@ Notable Changes
 - Local field potentials can now be recorded during simulations 
   :ref:`[Example] <sphx_glr_auto_examples_howto_plot_record_extracellular_potentials.py>`
 
-- Ability to optimize parameters to reproduce event related potentials from real data 
+- Ability to optimize parameters to reproduce event related potentials from real data,
```

Comment:
this is a change from previous version 0.2. You should add yours under "Current" version under the category "Changelog". 

I just realized that some folks have added theirs to the "Bug" section when it actually belongs to "Changelog" ... we'll fix that separately